install/yaml/install.yaml

```diff
@@ -20,7 +20,7 @@ metadata:
   namespace: agones-system
   labels:
     app: agones
-    chart: agones-1.1.0-rc
+    chart: agones-1.1.0
```

Comment:
same in this file

Reply:
Not until we've [completed the release](https://github.com/googleforgames/agones/blame/master/docs/governance/templates/release_issue.md#L42-L43). This will be the version for this release that gets pushed as a helm chart.